apps/balancer-tools/src/app/apr/(utils)/calculateApr.ts

```diff
@@ -2,17 +2,17 @@ import { Address } from "@bleu-balancer-tools/utils";
 import * as Sentry from "@sentry/nextjs";
 
 import * as balEmissions from "#/lib/balancer/emissions";
-import { pools } from "#/lib/gql/server";
+import { poolsWithoutCache } from "#/lib/gql/server";
```

Comment:
honestly I think it was my mistake having the default as cached. `#would` reverse this, so implement "poolsWithCache" and use it where needed, and leave the default not having cached.